app/pages/dashboard.vue

```diff
@@ -1,246 +1,214 @@
-                    class="text-blue-500">Create one now</NuxtLink>
-            </p>
-        </div>
-    </div>
-    <div class="flex flex-col px-4 xl:px-6 mt-8">
-        <h1 class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto uppercase font-bold mt-1">Actions</h1>
-        <ul class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto mt-2 text-center">
-            <li class="py-2 bg-emerald-700 text-white rounded hover:bg-emerald-600 transition-colors w-40 font-semibold uppercase"
-                style="font-size: smaller;">
-                <NuxtLink to="/forms/create">Create A Form</NuxtLink>
-            </li>
-        </ul>
-    </div>
-</template>
 <script setup lang="ts">
+import { DollarSignIcon, FileTextIcon, GitGraphIcon } from 'lucide-vue-next';
+import { h, type Component } from 'vue';
+
+
 definePageMeta({
-    middleware: ['auth']
-})
+  middleware: ["auth"],
+});
 
-const formCount = ref(0);
-const moneyCount = ref(0);
-const reachCount = ref(0);
+const stats = await useFetch<
+  APIResponse<{ forms: number; responses: number; earnings: number }>
+>("/api/v1/forms/me/stats", {
+  headers: {
+    Authorization: `Bearer ${getAuthToken()}`,
+  },
+  onResponseError({ response }) {
+    console.error(response);
+  },
+  onRequestError({ error }) {
+    console.error(error);
+  },
+})
+  .then(({ data }) => data.value?.body)
+  .catch((err) => {
+    return {} as any;
+  });
 
-const stats = await useFetch<APIResponse<{ forms: number, responses: number, earnings: number }>>("/api/v1/forms/me/stats", {
-    headers: {
-        Authorization: `Bearer ${getAuthToken()}`,
+const Card = defineComponent({
+  props: {
+    title: {
+      type: String,
+      required: true
     },
-    onResponseError({ response }) {
-        console.error(response)
+    count: Number,
+    icon: {
+      type: Object as PropType<Component>,
+      required: true
     },
-    onRequestError({ error }) {
-        console.error(error)
-    }
-}).then(({ data }) => data.value?.body).catch(err => { return {} as any; })
-formCount.value = stats?.forms || 0;
-moneyCount.value = stats?.earnings || 0;
-reachCount.value = stats?.responses || 0;
-const recents = await useFetch<APIResponse<{ forms: Drizzle.Form.select[] }>>(`/api/v1/forms/me/recents`, {
+    description: String
+  },
+  render() {
+    return h('div', [
+      h('div', { class: "flex items-center justify-between" }, [
+        h('div', [
+          h('p', { class: "font-bold text-xl uppercase" }, this.$props.title),
+          h('h3', { class: "text-2xl font-bold text-gray-900 mt-1" }, this.$props?.count || 0)
+        ]),
+        h('div', { class: "bg-gray-100 group-hover:bg-gray-200 rounded-full p-3 transition-colors" }, [
+          h(this.$props.icon, { class: "w-6 h-6 text-gray-700" })
+        ])
+      ]),
+      h('div', { class: "mt-4 flex items-center text-sm text-gray-600" }, this.$props.description)
+    ])
+  }
+})
```

Comment:
This is how to define a component within a component programmatically without having to create a new file